Mention the need to include icon (#9581)
gatsbyjs/gatsby: docs/docs/add-a-manifest-file.md

npm install --save gatsby-plugin-manifest
```

> 2.  Add the plugin to the `plugins` array in your `gatsby-config.js` file (don't forget to add the icon yourself as we don't supply one).

Reviewer: ```suggestion
2.  Add the plugin to the `plugins` array in your `gatsby-config.js` file (make sure to add the icon first!).
```

"as we don't supply one" may come off as rude 😅
